WR31/doormon.py

```diff
@@ -0,0 +1,159 @@
+"""
+Monitor the WR31 door enclosure
+"""
+############################################################################
+#                                                                          #
+# Copyright (c)2015, Digi International (Digi). All Rights Reserved.       #
+#                                                                          #
+# Permission to use, copy, modify, and distribute this software and its    #
+# documentation, without fee and without a signed licensing agreement, is  #
+# hereby granted, provided that the software is used on Digi products only #
+# and that the software contain this copyright notice,  and the following  #
+# two paragraphs appear in all copies, modifications, and distributions as #
+# well. Contact Product Management, Digi International, Inc., 11001 Bren   #
+# Road East, Minnetonka, MN, +1 952-912-3444, for commercial licensing     #
+# opportunities for non-Digi products.                                     #
+#                                                                          #
+# DIGI SPECIFICALLY DISCLAIMS ANY WARRANTIES, INCLUDING, BUT NOT LIMITED   #
+# TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A          #
+# PARTICULAR PURPOSE. THE SOFTWARE AND ACCOMPANYING DOCUMENTATION, IF ANY, #
+# PROVIDED HEREUNDER IS PROVIDED "AS IS" AND WITHOUT WARRANTY OF ANY KIND. #
+# DIGI HAS NO OBLIGATION TO PROVIDE MAINTENANCE, SUPPORT, UPDATES,         #
+# ENHANCEMENTS, OR MODIFICATIONS.                                          #
+#                                                                          #
+# IN NO EVENT SHALL DIGI BE LIABLE TO ANY PARTY FOR DIRECT, INDIRECT,      #
+# SPECIAL, INCIDENTAL, OR CONSEQUENTIAL DAMAGES, INCLUDING LOST PROFITS,   #
+# ARISING OUT OF THE USE OF THIS SOFTWARE AND ITS DOCUMENTATION, EVEN IF   #
+# DIGI HAS BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGES.                #
+#                                                                          #
+############################################################################

```

Comment:
The License on this does not match the license for the project itself.  I think this should have the minimal text for the MPL as is done in other projects (i.e. https://github.com/digidotcom/python-suitcase/blob/master/suitcase/fields.py#L1) rather than what is here.

As-is, the license on this file is incompatible with the MPL v2.0 as the MPL itself does not restrict the context in which the software may be used.